### PR TITLE
Add Tether (ERC20, Omni and Liquid)

### PR DESCRIPTION
**Pull request by chimp1984**
Based on https://github.com/bisq-network/bisq/pull/4434 from @pingiun

### CODE REVIEW

**Review by pingiun — DISMISSED** [review]

File: assets/src/main/java/bisq/asset/LiquidBitcoinAddressValidator.java
Comment (pingiun) [inline assets/src/main/java/bisq/asset/LiquidBitcoinAddressValidator.java]: this constructor should be removed, and the i18n key should be added to the super call in the default constructor

**Review by ripcurlx — APPROVED** [review]
utACK

I'll test this in more detail during release testing.

**Comment by wiz** [comment]
As per my last comment in PR #4434 for Tether USD, **any new assets now need to implement their own price feeds**. This PR should not have been merged without implementing a price feed to go with it, and now ops team has to scramble at the last minute to implement a pricefeed for USDT... in the future @bisq-network/bisq-maintainers please do not merge PRs for new assets unless they implement their own price feed in the pricenode code, and please also coordinate with the @bisq-network/pricenode-operators so they can upgrade and support the new feed before release time 😅

**Comment by pingiun** [comment]
@wiz the next release could still omit USDT right? I don't think we're in a rush to have it in the next client, so if the pricefeed code isn't ready these commits should just not be included in the next release...

**Comment by flix1** [comment]
Hi, this has now been stalled for a while. I believe it is important enough to try to push for it in the next release.

This issue has been in the works since January 2020. See previous proposals here https://github.com/bisq-network/proposals/issues/170 and here https://github.com/bisq-network/bisq/pull/4434

**Comment by wiz** [comment]
The reason that USDT was not added is because there are many USDT tokens on various blockchains / networks so it's not clear which one, and Bisq trade protocol has no way to support tokens that can be settled on different networks, also there was a concern raised that the price of USDT might differ across various USDT tokens if one network has issues etc.
https://wallet.tether.to/transparency

**Pull request merged**
